### ISSUE TITLE
.resize(true) ignores .height(value) and .width(value) after resize

### ISSUE THREAD

**Issue by azastupailo** [alexandersimoes/d3plus#545]
On the resize of the window viz ignores `.height()` and `.width()` methods.

For example in JSFiddle I set `.width(400)`:
https://jsfiddle.net/09xocev4/

Initial viz looks like (so `.width()` works as expected):
![image](https://cloud.githubusercontent.com/assets/6131645/18609631/d2f80212-7d06-11e6-98ec-4eca4b223f93.png)

After changing only the height of the window to bigger value it also increases the width of the viz:
![image](https://cloud.githubusercontent.com/assets/6131645/18609653/53ed25f0-7d07-11e6-9eb5-8b169a09d28e.png)

I think it should resize viz but change only height in this case.

**Comment by davelandry** [comment]
I agree, `.resize(true)` should not overwrite either dimension, but it currently overwrites both.

This will require storing a different pair of width and height variables, specific to the resize function, and then only using those sizes when the main width and/or height or not user-defined. @bryanph, are you available to work on your original resize code?

**Comment by davelandry** [comment]
Resolved in 2.0
